### ISSUE TITLE
Retrieval results

### ISSUE THREAD

**Issue by kimihailv** [mshukor/ViCHA#5]
Hello, in table 3 you compare ViCHa with ALBEF* (1.1M), the label of this table: "Zero-shot Comparison with SOTA on Flickr30K (after fine-tuning ViCHA on COCO) and COCO (pretrained model only)". Was ALBEF* fine-tuned on COCO before evaluation on Flickr30K too?

**Comment by mshukor** [comment]
Hello, yes, both are finetuned on COCO. They are comparable on  all benchmarks.
